easybuild/easyconfigs/l/LLVM/LLVM-3.7.0-intel-2015b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '3.7.0'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'cstd': 'gnu++11'}
+
+source_urls = ["http://llvm.org/releases/%(version)s"]
+sources = ["llvm-%(version)s.src.tar.xz"]
+
+builddependencies = [
+    ('CMake', '3.3.2'),
+    ('Python', '2.7.10'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON -DCMAKE_EXE_LINKER_FLAGS=-shared-intel '
```

Comment:
does `-DCMAKE_EXE_LINKER_FLAGS` pick up `$LDFLAGS`? If so, please use `="$LDFLAGS -shared-intel"`, if not, use `="-shared-intel"` (it's a bit hard to read without the quotes)